CMakeLists.txt

```diff
@@ -38,10 +39,13 @@ endif()
 
 if (GIT_PROGRAM AND CMAKE_FORMAT_PROGRAM)
   function (add_cmake_format_target name)
+    if (CMAKE_FORMAT_EXCLUDE AND ((CMAKE_GENERATOR STREQUAL "Unix Makefiles") OR (CMAKE_GENERATOR STREQUAL "Ninja")))
```

Comment:
Initially thought it would be better to be explicit with the tools that were tested, but I neglected to think about the rest of the ecosystems. Made the change.